src/flow/src/compute/state.rs

```diff
@@ -41,6 +42,14 @@ pub struct DataflowState {
 }
 
 impl DataflowState {
+    pub fn new_arrange(&mut self, name: Option<Vec<String>>) -> ArrangeHandler {
```

Comment:
This function does not use `DataflowState`, why use `&mut self`?

Reply:
adding it to dataflow now for debugging and monitoring purpose now